### PR TITLE
PG 15 updates

### PR DESCRIPTION
**Pull request by danieltprice**
PG 15 updates:
- Update extensions tables. A few extensions have a different version for PG 15.
- Update project creation steps to include selecting a PG version and a region.
- Compatibility table:
https://websitemain62807-dpricepg15updates.gtsb.io/docs/reference/compatibility/
- Project creation step (added selecting a region). The same change was made to guides that include creating a Neon project.
https://websitemain62807-dpricepg15updates.gtsb.io/docs/get-started-with-neon/setting-up-a-project/

### CODE REVIEW

File: content/docs/reference/compatibility.md [hunk 1]
```diff
@@ -19,37 +19,45 @@ Neon cloud service does not currently provide users with access permissions othe
...
+| hstore                   | [1.8](https://www.postgresql.org/docs/14/hstore.html)           | [1.8](https://www.postgresql.org/docs/15/hstore.html)            |                                                                                                                    |
+| intarray                 | [1.5](https://www.postgresql.org/docs/14/intarray.html)         | [1.5](https://www.postgresql.org/docs/15/intarray.html)          |                                                                                                                    |
+| isn                      | [1.2](https://www.postgresql.org/docs/14/isn.html)              | [1.2](https://www.postgresql.org/docs/15/isn.html)               |                                                                                                                    |
+| lo                       | [1.1](https://www.postgresql.org/docs/10/lo.html)               | [1.1](https://www.postgresql.org/docs/10/lo.html)                |                                                                                                                    |
```
Comment (bayandin) [inline content/docs/reference/compatibility.md:44]: I guess it's a typo, right?

```suggestion
| lo                       | [1.1](https://www.postgresql.org/docs/14/lo.html)               | [1.1](https://www.postgresql.org/docs/15/lo.html)                |                                                                                                                    |
```

Comment (danieltprice) [reply content/docs/reference/compatibility.md]: Yes. Thanks for that catch! Fixed.

File: content/docs/get-started-with-neon/setting-up-a-project.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ To set up a Neon project:
 
 1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
 2. If you are creating your very first project, click **Create the first project**. Otherwise, click **New Project**. 
-3. Enter a name for your project and click **Create Project**. Upon creating a project, you are presented with a dialog that provides the following information:
+3. Specify a name, a PostgreSQL version, and click **Create Project**. Upon creating a project, you are presented with a dialog that provides the following information:
```
Comment (lubennikovaav) [inline content/docs/get-started-with-neon/setting-up-a-project.md:9]: PostgreSQL version is optional.
Should we mention default here or it will be too wordy?

**Review by lubennikovaav — APPROVED** [review]
